app/assets/javascripts/shared/editor_toolbar.js

```diff
@@ -113,25 +114,25 @@ class EditorToolbar {
       var $inputElement = $(this),
           $toolbarElement = $inputElement.parent().prev(),
           $parentElement = $inputElement.parents('[data-behavior~=editor-field]'),
-          $scrollLimitElement = $parentElement.parents('.textile-wrap').parent(),
-          topOffset;
-      
+          $scrollLimitElement = $('[data-behavior~=textile-wrap]').length ? $('[data-behavior~=textile-wrap]') : $('[data-behavior~=view-content]'),
+          topOffset = $('[data-behavior~=textile-wrap]').length ? $parentElement.find('.editor-toolbar').outerHeight() : 85;
```

Comment:
What is `85` here? This smells like a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) and may be moved to a constant.

Reply:
fixed